src/hooks/useNotification.ts

```diff
@@ -0,0 +1,24 @@
+import { useContext } from 'react';
+import { NotificationContext } from '~/providers/NotificationProvider/NotificationProvider';
+import { TNotificationPayload } from '~/providers/NotificationProvider/types';
```

Comment:
i suggest to group these imports
```suggestion
import { NotificationContext, TNotificationPayload } from '~/providers';
```

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/2/commits/ba0c3959874ad8a573436f5456e5d6253b4026d2